client/src/pages/Energisers.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import Navbar from "./components/Navbar";
+import Footer from "./components/Footer.js";
+import EnergisersCard from "./components/EnergisersCard.js";
+import "./Energisers.css";
+// import NumberOfPages from "./components/NumberOfPages";
+import { useState, useEffect } from "react";
+
+function Energisers() {
+	const [allEnergizers, setAllEnergizers] = useState([]);
+	useEffect(() => {
+		fetch("/api/energizers")
```

Comment:
I was wondering for an hour trying to find  bug , because it wasn't working for me , and it was the spell of energizer , because my end point was /energiser so , I don't mind changing all them in my code to Z but we have to agree on something and stick to it in team ,because it might get complicated later as we carry on codding

Reply:
I think we should spell with Z as its more prevalent on the internet and even VSCODE highlights the s spelling as unknown word

Reply:
so I will change in my code and we should also edit page names